js/index.js

```diff
@@ -0,0 +1,29 @@
+// !! IMPORTANT README:
```

Comment:
Kindly remove the index.js file as there are only comments, not any code 😄 😉😄 😉.
thanks a lot, happy coding  😉.